dom0-updates/qubes-dom0-update

```diff
@@ -23,15 +23,15 @@ if [ "$1" = "--help" ]; then
 fi
 
 # Prevent template upgrade - this would override user changes
-TEMPLATE_EXCLUDE_OPTS="--exclude=`rpm -qa --qf '%{NAME},' qubes-template-\*`"
-PKGS=
-YUM_OPTS="$TEMPLATE_EXCLUDE_OPTS"
-GUI=
-CHECK_ONLY=
-ALL_OPTS="$TEMPLATE_EXCLUDE_OPTS $*"
-YUM_ACTION=
-QVMRUN_OPTS=
-CLEAN=
+[ -n "$TEMPLATE_EXCLUDE_OPTS" ] || TEMPLATE_EXCLUDE_OPTS="--exclude=`rpm -qa --qf '%{NAME},' qubes-template-\*`"
+[ -n "$PKGS" ] || PKGS=
+[ -n "$YUM_OPTS" ] || YUM_OPTS="$TEMPLATE_EXCLUDE_OPTS"
+[ -n "$GUI" ] || GUI=
```

Comment:
Yes, but not through environment. ;) It does no damage and looks more consistent. However, it's of course your call. If this PR makes sense at all... Not a pretty/great solution... https://github.com/QubesOS/qubes-issues/issues/2061#issuecomment-225429104